launch_ros/launch_ros/utilities/normalize_parameters.py

```diff
@@ -0,0 +1,185 @@
+# Copyright 2019 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Module with utility for normalizing parameters to a node."""
+
+from collections.abc import Mapping
+from collections.abc import Sequence
+import pathlib
+from typing import cast
+from typing import List  # noqa
+from typing import Optional
+from typing import Sequence as SequenceTypeHint
+from typing import Tuple  # noqa
+from typing import Union  # noqa
+from typing import Set  # noqa
+
+from launch.some_substitutions_type import SomeSubstitutionsType
+from launch.some_substitutions_type import SomeSubstitutionsType_types_tuple
+from launch.substitution import Substitution
+from launch.substitutions import TextSubstitution
+from launch.utilities import ensure_argument_type
+from launch.utilities import normalize_to_list_of_substitutions
+
+from ..parameters_type import ParameterFile
```

Comment:
This line seems to result in a `flake8` warning in the dev job: http://build.ros2.org/job/Ddev__launch_ros__ubuntu_bionic_amd64/21/consoleFull#console-section-17

@sloretz Can you please double check this.

Reply:
That's odd, this PR is from March. Is this a flake8 regression? `ParameterFile` is used in a type annotation. https://github.com/ros2/launch_ros/blob/2d5d3eac831788d54c41069378733837b48b44c8/launch_ros/launch_ros/utilities/normalize_parameters.py#L175


Maybe using the [PEP526 typing syntax](https://www.python.org/dev/peps/pep-0526/) will workaround the issue?

Reply:
The `dev` job users the Debian package of `flake8` which is different from the latest release we use on ci.ros2.org.